.github/workflows/release.yml

```diff
@@ -14,4 +14,4 @@ jobs:
     with:
       publishToBinaries: false
       mavenCentralSync: true
-      slackChannel: team-sonarlint-java
+      slackChannel: squad-sonar-solution
```

Comment:
Shouldn't it be `#squad-sonar-solution-bots`? In fact if it's just for the release, I'm fine having it in the discussion channel

Reply:
I think `#squad-sonar-solution-bots` is mostly for squad members to be aware of JIRA, Github and Cirrus events and so we don't polite our official `#squad-sonar-solution` where we have even external members (PMs, UXs, people from other squads). That said, I think these external members might be interested in getting the release notification (which won't happen often) so I think the `#squad-sonar-solution` channel might be good place.